### ISSUE TITLE
0.4.1: install target fails

### ISSUE THREAD

**Issue by kloczek** [openSUSE/libeconf#152]
```console
+ cd libeconf-0.4.1
+ DESTDIR=/home/tkloczko/rpmbuild/BUILDROOT/libeconf-0.4.1-2.fc35.x86_64
+ /usr/bin/cmake --install x86_64-redhat-linux-gnu
-- Install configuration: "RelWithDebInfo"
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/libeconf-0.4.1-2.fc35.x86_64/usr/lib64/pkgconfig/libeconf.pc
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/libeconf-0.4.1-2.fc35.x86_64/usr/lib64/libeconf.so.0.4.1
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/libeconf-0.4.1-2.fc35.x86_64/usr/lib64/libeconf.so.0
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/libeconf-0.4.1-2.fc35.x86_64/usr/lib64/libeconf.so
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/libeconf-0.4.1-2.fc35.x86_64/usr/include/libeconf.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/libeconf-0.4.1-2.fc35.x86_64/usr/include/libeconf_ext.h
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/libeconf-0.4.1-2.fc35.x86_64/usr/lib64/cmake/libeconf/libeconfTargets.cmake
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/libeconf-0.4.1-2.fc35.x86_64/usr/lib64/cmake/libeconf/libeconfTargets-relwithdebinfo.cmake
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/libeconf-0.4.1-2.fc35.x86_64/usr/lib64/cmake/libeconf/libeconfConfig.cmake
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/libeconf-0.4.1-2.fc35.x86_64/usr/lib64/cmake/libeconf/libeconfConfigVersion.cmake
-- Installing: /home/tkloczko/rpmbuild/BUILDROOT/libeconf-0.4.1-2.fc35.x86_64/usr/bin/econftool
-- Set runtime path of "/home/tkloczko/rpmbuild/BUILDROOT/libeconf-0.4.1-2.fc35.x86_64/usr/bin/econftool" to ""
CMake Error at x86_64-redhat-linux-gnu/doc/cmake_install.cmake:46 (file):
  file INSTALL cannot find
  "/home/tkloczko/rpmbuild/BUILD/libeconf-0.4.1/x86_64-redhat-linux-gnu/doc/html":
  No such file or directory.
Call Stack (most recent call first):
```
```console
[tkloczko@barrel x86_64-redhat-linux-gnu]$ cmake -L
CMake Warning:
  No source or binary directory provided.  Both will be assumed to be the
  same as the current working directory, but note that this warning will
  become a fatal error in future CMake releases.


CMake Error: The source directory "/home/tkloczko/rpmbuild/BUILD/libeconf-0.4.1/x86_64-redhat-linux-gnu" does not appear to contain CMakeLists.txt.
Specify --help for usage, or press the help button on the CMake GUI.
-- Cache values
BASH_PROGRAM:FILEPATH=/usr/bin/bash
BUILD_DOCUMENTATION:BOOL=ON
BUILD_TESTING:BOOL=ON
CMAKE_BUILD_TYPE:STRING=RelWithDebInfo
CMAKE_C_FLAGS_RELWITHDEBINFOSTRICT:STRING=-O2 -g -Werror -W -Wall -DXTSTRINGDEFINES -D_FORTIFY_SOURCE=2 -fstack-protector-strong -funwind-tables -fasynchronous-unwind-tables -fstack-clash-protection -Werror=return-type -flto=8 -Wbad-function-cast -Wcast-align -Wcast-qual -Winline -Wmissing-declarations -Wmissing-prototypes -Wnested-externs -Wshadow -Wstrict-prototypes -Wundef
CMAKE_C_FLAGS_SANITIZEADDRESS:STRING=-O2 -g -Wall -fsanitize=address -fno-optimize-sibling-calls -fsanitize-address-use-after-scope -fno-omit-frame-pointer
CMAKE_INSTALL_PREFIX:PATH=/usr
CMAKE_LINK_FLAGS_SANITIZEADDRESS:STRING=-fsanitize=address
```

**Comment by schubi2** [comment]
The first one I have fixed it with version 0.4.2. Thanks for the report !
The second one I cannot reproduce it because I do not really know your environment.
At least with this calls it should work:
https://github.com/openSUSE/libeconf/blob/master/INSTALL.md

**Comment by kloczek** [comment]
Just tested and looks like 0.4.2 is OK.
Thank you.
